reporters_db/data/reporters.json

```diff
@@ -18152,15 +18211,22 @@
                 "Ridgely's Notebook V, 362",
                 "Ridgely's Notebook III, 5",
                 "Ridgely's Notebook II, 180",
-                "Ridgely's Notebook I, 244"
+                "Ridgely's Notebook I, 244",
+                "Ridgely’s Notebook IV, 33",
+                "Ridgely’s Notebook V, 362"
             ],
             "mlz_jurisdiction": [
                 "us:de;"
             ],
             "name": "Ridgely's Notebook",
             "notes": "Dates are rough, but wide, estimated from a series called Delaware Cases, which had these dates. See: http://www.worldcat.org/oclc/2882722",
             "variations": {
-                "Ridgely's Notebook I": "Ridgely's Notebook"
+                "Ridgely's Notebook I": "Ridgely's Notebook",
+                "Ridgely’s Notebook I": "Ridgely's Notebook",
+                "Ridgely’s Notebook II": "Ridgely's Notebook II",
+                "Ridgely’s Notebook III": "Ridgely's Notebook III",
+                "Ridgely’s Notebook IV": "Ridgely's Notebook IV",
+                "Ridgely’s Notebook V": "Ridgely's Notebook V"
```

Comment:
again here.. are these variations or are we normalizing.

Reply:
This are variations with unicode characters, in this specific case I think i can replace it with a regex with an optional apostrophe or unicode quotation mark to avoid having many variations, like the proposed solution here: https://github.com/freelawproject/eyecite/issues/121#issuecomment-1228820241

Reply:
yeah ... this is right.